README.md

```diff
@@ -11,16 +11,82 @@ This repo should be used as a template when creating new repos for the Apprentic
+| das-apprentice-commitments-web | https://github.com/SkillsFundingAgency/das-apprentice-commitments-web |
+| das-apprentice-portal-web | https://github.com/SkillsFundingAgency/das-apprentice-portal-web |
+| das-apprentice-accounts-web | https://github.com/SkillsFundingAgency/das-apprentice-accounts-web |
+| das-apprentice-accounts-api | https://github.com/SkillsFundingAgency/das-apprentice-accounts-api |
+| das-apprentice-commitments-jobs | https://github.com/SkillsFundingAgency/das-apprentice-commitments-jobs |
+
+| Configuration | Repo |
+| ------ | ------ |
+| das-employer-config | https://github.com/SkillsFundingAgency/das-employer-config |
+| das-employer-config-updater | https://github.com/SkillsFundingAgency/das-employer-config-updater |
+
+
+das-apprentice-portal-web solution includes the Apprentice Commitments homepage, SharedUI nuget components which are consumed by other microservicews, as well as an authentication component.
 
 ## Developer Setup
 
+The portal uses azure storage to retrieve settings. To use local settings UseDevelopmentStorage=true should be set in the local appsettings.deverlopment.json file as detailed below. Repo das-employer-config section [das-apprentice-portal-web](https://github.com/SkillsFundingAgency/das-employer-config/tree/master/das-apprentice-portal-web) holds the settinge for the portal web.
+
+After cloning the portal repo it should build without any issues.  To run the portal web home page requires a running login service, commitments website & accounts website.  The config section 'ApplicationUrls' indicates where the service should point to (eg with the login service, locally https://localhost:5001 or https://login.at-aas.apprenticeships.education.gov.uk/ for the at environment ).  If you want to run all locally, then the login service, accounts website and commitments website all need to be running locally too. 
+
 ### Requirements
 
 ### Setup
 
-### Config
\ No newline at end of file
+### Config
+
+The portal webiste hosts the homepage and gets its configuration settings from an appsettings.json file.
+Create a local appsettings.development.json file as follows:
+
+    {
+      "Logging": {
+        "LogLevel": {
+          "Default": "Information",
+          "Microsoft": "Warning",
+          "Microsoft.Hosting.Lifetime": "Information"
+        }
+      },
+      "AllowedHosts": "*",
+      "ConfigurationStorageConnectionString": "UseDevelopmentStorage=true",
+      "ConfigNames": "SFA.DAS.ApprenticePortal.Web",
+      "EnvironmentName": "LOCAL",
+      "Cdn": {
+        "Url": "https://das-at-frnt-end.azureedge.net"
+      },
+      "GoogleAnalytics": {
+        "GoogleTagManagerId": "GTM-MP5RSWL"
+      },
+      "ApprenticePortalOuterApi": {
+        "ApiBaseUrl": "https://localhost:5123/",
+        "SubscriptionKey": "key",
+        "ApiVersion": "1"
+      },
+      "ApplicationUrls": {
+        //"ApprenticeHomeUrl": "https://portal.at-aas.apprenticeships.education.gov.uk/",
```

Comment:
It's worth removing the commented lines

Reply:
These were left is as a dev may want to point to AT and not local.. removed now